CHANGELOG.md

```diff
@@ -427,6 +458,42 @@ Added
   metadata for a package by extracting library requirement information from the
   binary ELF files in the package. (#59569)
 
+- Sign authentication replies to prevent MiTM (cve-2020-22935)
```

Comment:
this looks to be under the wrong section 3003 which was released a while ago. I think these just need to be removed